src/content/language/functions.md

```diff
@@ -266,6 +266,41 @@ assert(loudify('hello') == '!!! HELLO !!!');
 This example uses an anonymous function.
 More about those in the next section.
 
+## Function types
+
+We can specify the type of a function, which is known as a *function type*.
+It is actually the type of a first-class function object: An expression whose 
+type is a function type can only evaluate to a first-class function object.
```

Comment:
These two sentences are difficult for me to parse. What do you think about something like "A function type consists of a return type (including `void`) and a list of zero or more parameters."?

Reply:
Lines 272 and 273 describe the semantics of a function type, that is, what it means. Perhaps there's a need to define 'first-class function object', but that's probably difficult to cram into a few words.

A sentence like "A function type consists of ..." describes the syntax of the function type.

So why don't we just have both?

This would fit quite nicely with the next sentence where the syntax is described in a more indirect fashion.